services/static-webserver/client/compile.json

```diff
@@ -136,7 +136,7 @@
       "class": "osparc.Application",
       "theme": "osparc.theme.products.tis.ThemeDark",
       "name": "tis",
-      "title": "TI Plan - IT'IS",
+      "title": "TIP V3.0 - IT'IS",
```

Comment:
Q: if the display_name from the db is providing these,  what are these values in `compile.json` for?

Reply:
These are used for compiling the different products, the different source-outputs, at that point the display_name is not available. It is mainly used for populating some metadata that goes to the index.html and tabs...